package.json

```diff
@@ -43,6 +44,7 @@
   "devDependencies": {
     "@appium/eslint-config-appium": "^6.0.0",
     "@semantic-release/git": "^10.0.1",
+    "@semantic-release/changelog": "^6.0.1",
```

Comment:
I assume this should go before `"@semantic-release/git"` in alphabetic order. Same comment to other PRs